code/game/objects/items/devices/taperecorder.dm

```diff
@@ -430,6 +433,26 @@
 		to_chat(user, span_notice("You wind the tape back in."))
 		respool()
 
+/obj/item/tape/AltClick(mob/user)
+	. = ..()
+	if (firstFlip)
+		to_chat(usr, "<span class='notice'>You flip the tape so you can record on the clean magnetic strip.</span>")
+		firstFlip = FALSE
+	else
+		to_chat(usr, "<span class='notice'>You flip the tape back around.</span>")
+	tapeflip()
+
+/obj/item/tape/verb/flipVerb()
```

Comment:
No more verbs, if you're up for it you can use a contextual screentip instead--you will want to check out https://github.com/tgstation/tgstation/tree/master/code/datums/elements/screentips#datumelementcontextual_screentip_bare_hands.